app/controllers/ideas_controller.rb

```diff
@@ -5,12 +5,15 @@ class IdeasController < ApplicationController
   # GET /ideas
   # GET /ideas.json
   def index
-    if params[:view].eql?('assigned')
-      @ideas = Idea.where(assigned_user_id: [current_user.id])
-    else
-      @ideas = Idea.all
+    @ideas = case params[:view]
+      when 'assigned'
+        Idea.where(assigned_user_id: [current_user.id])
+      when 'submitted'
+        Idea.where.not(submission_date: [nil, ''])
+      else
+        Idea.all
+      end
     end
-  end
 
```

Comment:
Sorry for pinching this piece of work from you Katharine, I was just having a look at the active record queries as I haven't done any yet then started playing around with this.  What are peoples thoughts on the best way to structure a controller with multiple params? I started with the if else if but I think it looks a bit clearer like this, any thoughts? Cheers